world_universities_and_domains.json

```diff
@@ -129113,5 +129128,451 @@
       "ulb.be"
     ],
     "country": "Belgium"
+  },
+  {
+    "web_pages": [
+      "https://cieep.com/"
+    ],
+    "name": "Progresa",
+    "alpha_two_code": "ES",
+    "state-province": null,
+    "domains": [
+      "cieep.com"
+    ],
+    "country": "Spain"
+  },
+  {
+    "web_pages": [
+      "https://www.merito.pl/"
+    ],
+    "name": "University WSB Merito",
+    "alpha_two_code": "PL",
+    "state-province": null,
+    "domains": [
+      "student.wroclaw.merito.pl"
+    ],
+    "country": "Poland"
+  },
+  {
+    "web_pages": [
+      "https://eui.edu.eg/"
+    ],
+    "name": "Egypt University of Informatics",
+    "alpha_two_code": "EG",
+    "state-province": null,
+    "domains": [
+      "students.eui.edu.eg"
+    ],
+    "country": "Egypt"
+  },
+  {
+    "web_pages": [
+      "https://www.upchiapas.edu.mx/"
+    ],
+    "name": "Universidad Politecnica de Chiapas",
```

Comment:
If the university's name is from their native language. I prefer to use the exact symbol from the name.

For this case, should be `Universidad Politécnica de Chiapas`

Reply:
The same thing applies to non-English university name 😄

Reply:
Ok, I'll be changing those